at_commons/lib/src/exception/at_exception_manager.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:at_commons/at_commons.dart';
+
+class AtExceptionManager {
+  static AtException createException(AtException atException) {
+    atException.message = atException.getTraceMessage();
+    return atException;
```

Comment:
What's the expected usage of this? And should it be renamed to 'createExceptionWithFullTraceMessage' or something similar, for clarity?

Reply:
The createException takes atException and check if its is an instance of AtClientException. If yes, returns as is. Else converts the exception to AtClientException and returns the exception back